__tests__/Hospital-Detail /doctor/DoctorCard.test.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import DoctorCard from "./../../../components/doctor/DoctorCard";
+
+const mockDoctor = {
```

Comment:
This mock data should be imported from the mocks folder

Reply:
since the doctor sample that am currently using and the one in the mock folder are different so i can't import it.